src/NuGet.Core/NuGet.Frameworks/PublicAPI/net40/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
+NuGet.Frameworks.NuGetFramework.NuGetFramework(string frameworkIdentifier, System.Version frameworkVersion, string frameworkProfile) -> void
```

Comment:
Should this be in shipped? :D 
Given that it's a regression?

I think it's ok if it stays in unshipped for now...and alter it gets fixed, it's really on Andy and how pedantic he wants to be.

Reply:
I'd prefer it to be in shipped, rather than unshipped, but fixing a breaking change is more important, so I won't block the PR.

Reply:
agree. Done 👍🏻